components/layout/Header.tsx

```diff
@@ -21,15 +21,15 @@ export const Header = () => {
               <Link href={"/"} passHref>
                 <a className={styles.navitem}>
                   <SearchIcon />
-                  Hae
+                  <span className={styles["navitem-text"]}>Hae</span>
```

Comment:
I'm just gonna ask now so I don't need to speculate - why not the same `styles.navitem-text` syntax that was in the previous commits?

Reply:
It's because of JavaScript; object keys can't be written with `-` with the dot notation, and  I think it's clearer to write css-classes with dash (so, `.navitem-text`). So, when I want to access the value of the key `navitem-class`,  I need to use that different syntax to be able access it. This is purely my opinion on how to write CSS, so it's definitely discussable 😄